lib/src/ref_name.rs

```diff
@@ -95,7 +95,7 @@ pub struct RemoteName(str);
 /// Use `.as_str()` or `.as_symbol()` for displaying. Other than that, this can
 /// be considered an immutable `String`.
 // Eq, Hash, and Ord must be compatible with WorkspaceName.
-#[derive(Clone, ContentHash, Debug, Eq, Hash, Ord, PartialEq, PartialOrd)]
+#[derive(Clone, ContentHash, Debug, Eq, Hash, Ord, PartialEq, PartialOrd, serde::Serialize)]
 pub struct WorkspaceNameBuf(String);
```

Comment:
insert `#[serde(transparent)]` (thought it's noop for JSON)
https://users.rust-lang.org/t/serialize-struct-with-one-field-as-inner-field/110482/2

Can you also add `serde::Serialize` to `WorkspaceName` type for consistency?